bedrock/firefox/templates/firefox/developer/whatsnew-fx68-trailhead.html

```diff
@@ -0,0 +1,25 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_66" %}
```

Comment:
This has the lang file from 66, is that okay?

Reply:
Good catch!